views/albums.hbs

```diff
@@ -0,0 +1,9 @@
+ <body id="albums">
+ {{#each response}} 
+  <div class="albums-div-info">
+  <img class="albums-image" src="{{this.images.0.url}}" alt="{{this.name}}" />
+    <h2 class="albums-name">{{this.name}}</h2>
+    <a href="/albums/:artistId">View Tracks</a>
```

Comment:
You're sending the user to the `albums` page again instead of sending to `tracks` also you have `:artistId` that syntax is for the route in the `app.js` file, here in the `hbs` file you should pass the actual `id`:
```
<a href="/tracks/{{id}}">View Tracks</a>
```